test/drenv/envfile.py

```diff
@@ -3,9 +3,31 @@
 
 import os
 import copy
+import platform
 
 import yaml
 
+VM = "$vm"
+CONTAINER = "$container"
+SHARED_NETWORK = "$network"
+
+
+def platform_defaults():
+    # By default, use minikube defaults.
+    d = {VM: "", CONTAINER: "", SHARED_NETWORK: ""}
+
+    operating_system = platform.system().lower
+    if operating_system == "linux":
+        d[VM] = "kvm2"
+        d[CONTAINER] = "podman"
+        d[SHARED_NETWORK] = "default"
+    elif operating_system == "darwin":
+        d[VM] = "hyperkit"
+        d[CONTAINER] = "podman"
```

Comment:
Your call on using docker or podman on macOS. Since podman is experimental, I guess docker is a better choice but I did not try any of them on macOS.